avm/client/src/index.ts

```diff
@@ -82,28 +80,30 @@ class HostImportsConfig {
     }
 }
 
-const interpreter_wasm = toByteArray(wasmBs64);
-
-/// Instantiates WebAssembly runtime with AIR interpreter module
-async function interpreterInstance(cfg: HostImportsConfig, logFunction: LogFunction): Promise<Instance> {
-    /// Create host imports that use module exports internally
+// Instantiates WebAssembly runtime with AIR interpreter module
+async function interpreterInstance(
+    module: WebAssembly.Module,
+    cfg: HostImportsConfig,
+    logFunction: LogFunction,
+): Promise<Instance> {
+    // Create host imports that use module exports internally
```

Comment:
Suggest to start non-doc comment with a lowercase letter, this convention is used in the Rust part.